core/t_set.go

```diff
@@ -0,0 +1,310 @@
+package core
+
+import (
+	"errors"
+	"sort"
+)
+
+func evalSADD(args []string, store *Store) []byte {
+	if len(args) < 2 {
+		return Encode(errors.New("ERR wrong number of arguments for 'SADD' command"), false)
```

Comment:
Please use standardized error builders which were recently introduced to return errors. Same for all other occurrences.

Reply:
Hi @JyotinderSingh, I have updated the code with new error standards. I had to add a new error type as the existing one was not per the Redis SET data commands standard. Please review and merge if everything looks good.